samples/templates/default-azuredeploy.json

```diff
@@ -292,13 +292,13 @@
                 {
                     "apiVersion": "2020-12-01",
                     "type": "siteextensions",
-                    "name": "AspNetCoreRuntime.7.0.x64",
+                    "name": "AspNetCoreRuntime.8.0.x64",
```

Comment:
Do you still use this template? We moved over to AppService Docker deployments which simplifies this a lot. The tags also make it really easy to go forward/back versions too